examples/webgl_loader_fbx.html

```diff
@@ -31,19 +31,33 @@
 			import { OrbitControls } from 'three/addons/controls/OrbitControls.js';
 			import { FBXLoader } from 'three/addons/loaders/FBXLoader.js';
 			import { GUI } from 'three/addons/libs/lil-gui.module.min.js';
+			import { EXRLoader } from 'three/addons/loaders/EXRLoader.js';
+			import { RGBELoader } from 'three/addons/loaders/RGBELoader.js';
+			import { TGALoader } from 'three/addons/loaders/TGALoader.js';
+			import { TIFFLoader } from 'three/addons/loaders/TIFFLoader.js';
+			import { DDSLoader } from 'three/addons/loaders/DDSLoader.js';
 
-			let camera, scene, renderer, stats, object, loader, guiMorphsFolder;
-			const clock = new THREE.Clock();
+			const manager = new THREE.LoadingManager();
 
+			manager.addHandler( /\.tga$/i, new TGALoader() );
+			manager.addHandler( /\.dds$/i, new DDSLoader() );
+			manager.addHandler( /\.exr$/i, new EXRLoader() );
+			manager.addHandler( /\.hdr$/i, new RGBELoader() );
+			manager.addHandler( /\.tiff$/i, new TIFFLoader() );
```

Comment:
Um, now that I look at the code I'm not so happy that the FBX example now loads all the modules by default. In production, that makes the example considerably slower to load.

 How about we remove the modifications to `webgl_loader_fbx` and just accept that we don't have a test asset for non-native textures?

Reply:
I'll try to make a checkbox in UI that when checked will asynchronously import  the loaders. Let's see (if I manage to do that) how would you like it that way.

Reply:
Wouldn't it be more simple to test for `with_non_native_textures` and then asynchronously load the texture loaders when the asset is selected?

Reply:
Better idea indeed. import the loaders when selecting the `with_non_native_textures.fbx` then when they're ready import the asset itself. Will try to do that.

Reply:
I pushed a new commit, now lazily importing loaders.
Note: I had to increase ESLint ECMA version in order to not complain about import() used as a function.

Reply:
> Note: I had to increase ESLint ECMA version in order to not complain about import() used as a function.

This is unfortunately not possible since we have to stick to ECMA 2018. Please revert the changes to `.eslintrc.json`.

Sorry, I think it's best to remove the asset from the example. It makes things just too complicated.

Reply:
I understand. I reverted it. I let there though some improvements on `webgl_loader_fbx.html` that happened during this PR (including some changes from `mrdoob`).

Regarding testing, if there would be a possibility to unit-test the assets we won't need to pile up to HTML examples.
However that would only be possible if in unit-tests we could leverage a mock server (something like [Sinon](https://sinonjs.org/) or [MSW](https://mswjs.io/) ) so that we could load assets from disk during unit tests.

Maybe I will come up with such PR at some point.
Do you a feature like that in unit tests would be welcomed?